homeassistant/components/jellyfin/client_wrapper.py

```diff
@@ -34,22 +33,19 @@ async def validate_input(
     return userid
 
 
-def create_client() -> JellyfinClient:
+def create_client(device_id: str, device_name: str | None = None) -> JellyfinClient:
     """Create a new Jellyfin client."""
-    jellyfin = Jellyfin()
-    client = jellyfin.get_client()
-    _setup_client(client)
-    return client
+    if device_name is None:
+        device_name = socket.gethostname()
```

Comment:
I was worried this might do I/O but it just does `uname()` under the hood on linux
`strace -f -s4096 -o /root/as python3 -c 'import socket; socket.gethostname()'`

Reply:
I intend to moclve that to using HA location name from config